packages/gatsby-theme-styleguide/src/gatsby-components/CalloutVue/CalloutVue.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import styled from 'styled-components'
+import { DisplayExternalLink16 } from '@mozaic-ds/icons/react'
+
+const VueJSBlock = styled.div`
+  display: flex;
+  align-items: center;
+  background-color: '#25a8d0';
+  border: 1px solid rgb(37, 168, 208);
+  padding: 1.375rem 1.5rem;
+  color: rgb(30, 95, 130);
+  font-size: 1rem;
+`
+
+const Link = styled.a`
+  display: flex;
+  font-weight: bold;
+  margin: 0 0.625rem;
+`
+
+const VueIcon = styled.svg`
+  margin-left: 0.5rem;
+  margin-right: 0.25rem;
+`
+
+const CalloutVue = ({ link }) => {
+  return (
+    <VueJSBlock>
+      <b>Available on: </b>
```

Comment:
can we make this link clickable?

Reply:
@nherwegh 
Sure we can, what should be clickable? 
The full block or the full text? 
rn only the display-external-link icon is clickable

Reply:
the great thing would be to make: 
"VueJS" & the icon clickable